models/chef/Chef.validation.js

```diff
@@ -0,0 +1,16 @@
+import Joi from 'joi';
+import { addressValidationSchema } from '../order/order.validation';
```

Comment:
why do we use address schema from order? if it is common it should be placed in common folder to be clear for all